app/kubernetes-ingress-controller/1.0.x/deployment/k4k8s-enterprise.md

```diff
@@ -137,13 +144,28 @@ $ helm repo update
 # Helm 2
 $ helm install kong/kong \
     --name demo --namespace kong \
-    --values https://l.yolo42.com/k4k8s-enterprise-helm-values
+    --values values.yaml
 
 # Helm 3
 $ helm install kong/kong --generate-name
     --namespace kong \
-    --values https://l.yolo42.com/k4k8s-enterprise-helm-values \
-     --set ingressController.installCRDs=false
+    -f values.yaml \
+    --set ingressController.installCRDs=false
+```
+
+### Example values.yaml
+```
+image:
+  repository: kong-docker-kong-enterprise-k8s.bintray.io/kong-enterprise-k8s
+  tag: 1.3.0.0-alpine
+  pullSecrets:
+  - kong-enterprise-k8s-docker
```

Comment:
Retroactive updates to old docs are confusing. Between the new 2.3 registry and Bintray's impending demise, not sure what'd be best here. This works and lines up with the rest of the doc (the pull secret used here also changed; its name no longer matched what we created earlier) for the purpose of this PR, while recognizing that we have a larger revamp planned in DOCS-1605:

```suggestion
  repository: kong-docker-kong-enterprise-edition-docker.bintray.io/kong-enterprise-edition
  tag: 2.2.1.0-alpine
  pullSecrets:
  - kong-enterprise-edition-docker
```

Reply:
I did pull these values directly  from the URL that was previously mentioned in the docs, happy to update these 😄